extensions/amp-form/0.1/amp-form.js

```diff
@@ -440,17 +440,20 @@ export class AmpForm {
    */
   doXhr_(opt_extraValues) {
     const isHeadOrGet = this.method_ == 'GET' || this.method_ == 'HEAD';
+    const values = this.getFormAsObject_(opt_extraValues);
+    this.renderTemplate_(values);
```

Comment:
this will render all templates which causes unnecessary rendering of success/failure templates  as well. would be nice if `renderTemplate` could take the type of target template.